recipes/datashape/bld.bat

```diff
@@ -0,0 +1,2 @@
+"%PYTHON%" setup.py install 
```

Comment:
For simple python-only builds, it's preferable to not use `bld.bat` or 'build.sh` and instead use the following in the `build` section of the `meta.yaml`:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L26